lighthouse-core/audits/manifest-short-name.js

```diff
@@ -28,6 +28,9 @@ class ManifestShortName extends Audit {
       category: 'Manifest',
       name: 'manifest-short-name',
       description: 'Manifest contains `short_name`',
+      helpText: 'Add an optional `short_name` to ensure that your app\'s ' +
```

Comment:
FWIW, as I mentioned in https://github.com/GoogleChrome/lighthouse/issues/23#issuecomment-270461938, the "optional" part of this can get confusing. In some contexts it is optional (`name` is just used in its stead), but for add to homescreen, it appears `short_name` is required. So it's optional depending on what the goal of the audit is.